### PR TITLE
fix: security badge link

### PR DESCRIPTION
**Pull request by mallachari**
Fixes: https://github.com/stoplightio/elements/issues/1333

Adds optional `allowRouting` prop to Docs that we can use with API component and which defaults to `false` for external usage.

### CODE REVIEW

**Review by philsturgeon — APPROVED** [review]

File: packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.spec.tsx [hunk 1]
```diff
@@ -115,17 +115,33 @@ describe('HttpOperation', () => {
...
       );
       const apikeyBadge = getSecurityBadge(/API Key/i);
       expect(apikeyBadge?.closest('a')).toHaveAttribute('href', '/reference/todos/openapi.v1.json?security=api_key');
 
       unmount();
     });
+
+    it('should not contain link to Overview for operation with uri by default', () => {
+      const { unmount } = render(
+        <Router>
```
Comment (mpodlasin) [inline packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.spec.tsx:136]: So shouldn't we be able to remove `Router` wrapper from that test, to really make sure it works without react router?

Comment (mallachari) [reply packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.spec.tsx:136]: Good point!

Comment (mallachari) [reply packages/elements-core/src/components/Docs/HttpOperation/HttpOperation.spec.tsx]: Updated

**Review by mpodlasin — APPROVED** [review]
🚀

**Pull request merged**